lib/osproject_android.rb

```diff
@@ -1,18 +1,30 @@
 require_relative 'osproject'
 require_relative 'osproject_helpers'
+require_relative 'osnetwork'
 
 class OSProject::GoogleAndroid < OSProject
   attr_accessor :app_class_location
 
   def initialize(app_class_location, os_app_id)
     @app_class_location = app_class_location
-
+    
     directory_split = app_class_location.split('/', -1)
-    lang = directory_split[-1].split(".")[1]
+    lang_array = directory_split[-1].split(".")
+
+    if lang_array.length() == 1
+      puts 'Missing Application class file extension (.kt or .java)'
+      error_track_message = "error=user missed Application class file extension;"
+      NetworkHandler.instance.send_track_actions(os_app_id, 'android', "nil", error_track_message)
```

Comment:
You should be able to use `nil` using string interpolation when we generate the header.